pinot-controller/src/main/resources/app/components/Homepage/Operations/RebalanceServer/RebalanceServerResponses/RebalanceServerRebalanceSummaryResponse.tsx

```diff
@@ -0,0 +1,35 @@
+import {Grid} from "@material-ui/core";
+import {RebalanceServerSection} from "../RebalanceServerSection";
+import React from "react";
+import './CustomCodeMirror.css';
+import {RebalanceServerResponseCard} from "./RebalanceServerResponseCard";
+import {RebalanceServerSectionResponse} from "./RebalanceServerSectionResponse";
+
+export const RebalanceServerRebalanceSummaryResponse = ({ response }) => {
+    const responseSectionsToShow = [
+        {
+            name: 'I. Server Information',
+            key: 'serverInfo'
+        },
+        {
+            name: 'II. Segment Information',
+            key: 'segmentInfo'
+        }
+    ];
+
```

Comment:
@somandal for adding new sections inside Rebalance Summary Section  you just need to provide the `key` and `name of the section` and it will render

Reply:
cc @J-HowHuang, once the Tenant info changes are in, we'll want to add it here so please make a note of this